megamek/src/megamek/client/generator/RandomGenderGenerator.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2020 - The MegaMek Team. All Rights Reserved.
+ *
+ * MegaMek is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MegaMek is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MegaMek. If not, see <http://www.gnu.org/licenses/>.
+ */
+package megamek.client.generator;
+
+import megamek.common.Compute;
+import megamek.common.enums.Gender;
+
+/**
+ * Author's Note: This is for generating Biological Gender (strictly speaking, the term is Sex) only,
+ * with the other-style generation being implemented for future gender identity use
+ */
+public class RandomGenderGenerator {
+    //region Variable Declarations
+    private static int percentFemale = 50;
+    //endregion Variable Declarations
+
+    //region Constructors
+    public RandomGenderGenerator() {
+
+    }
+    //endregion Constructors
+
+    //region Getters/Setters
+    public static int getPercentFemale() {
+        return percentFemale;
+    }
+
+    public static void setPercentFemale(int i) {
+        percentFemale = i;
```

Comment:
Do we need sanity checks here?

Reply:
Hmm, the only chance of this not fitting between the standard limits is that the person manually edited the save file, and even then it will generate properly based on their input value (although the options might not load because of an out of bounds error)

I'm leaning towards saying no